elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -150,49 +156,177 @@ $LOCAL-VARIABLES: (
+    @include browser-query(ie11) {
+      border-bottom: 1px solid transparent;
+    }
+  }
+  @at-root #{&}([pfe-variant="wind"][aria-selected="false"]:last-of-type):hover {
+    @include browser-query(ie11) {
+      border-bottom: #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+}
+
+:host{
+  @at-root #{&}([vertical][pfe-variant="wind"]){
+    --pfe-tabs__tab--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
+    --pfe-tabs--TextAlign: left !important;
+    @include browser-query(ie11) {
+      border-left: #{pfe-fetch(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-fetch(surface--border)};
+    }
+  }
+
+
+
+
+  @at-root #{&}([pfe-variant="wind"][aria-selected="false"]:hover){
+    --pfe-tabs__tab--BorderBottom: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    @include browser-query(ie11) {
+      border-bottom:  #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="true"]){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+    --pfe-tabs__tab--BorderLeft:  #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+    padding-left:                 calc(#{pfe-var(container-padding)} * 1.5 - 2px);
+    @include browser-query(ie11) {
+      border-right:  0;
+      border-left:   #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(highlight)};
+      padding-left:  10px;
+      border-bottom: 0 !important;
+    }
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]:hover){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+    --pfe-tabs__tab--BorderBottom: 0;
+    --pfe-tabs__tab--BorderLeft: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    padding-left: calc(#{pfe-var(container-padding)} * 1.5 - 2px);
+    @include browser-query(ie11) {
+      border-right:  0;
+      border-bottom: 0;
+      padding-left:  10px;
+      border-left:   #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+}
+
+/// ===========================================================================
+/// WIND TAB VARIANTS / saturated
+/// ===========================================================================
+
+:host {
+  @at-root #{&}([on="saturated"][pfe-variant="wind"]) {
+    color: #{pfe-var(link--focus--on-dark)};
```

Comment:
```suggestion
    color: #{pfe-var(text--focus--on-saturated)};
```

Reply:
@starryeyez024 This gave me a dark grey color and was hard to read on saturated BG's. I used `ui-base--hover--on-saturated` instead. Let me know if that works.

Reply:
ah my bad, this was a combination of me first suggesting we use the --on-saturated version of the link var, but then i went back and changed to to "text" which doesn't have a focus state! 

You'll likely want to use one of these:
`text--on-saturated` or `text--muted--on-saturated`